### PR TITLE
Introduce separate issue templates for bugs and feature requests

### LINKED ISSUE #5953: Multiple issue templates
Suggestion.

I just noticed that GitHub allows choosing between issue templates.

To see what I mean, go here and click 'New issue': https://github.com/Fody/Fody/issues

I think that would be useful for us.

For example:

- Bug
- Suggestion

Any others?

### PR DESCRIPTION
**Pull request by drewnoakes**
Fixes #5953

## Proposed changes

- Split the issue template into two separate templates
  - Bugs / issues
  - Feature proposals

## Test methodology

- Untested. Pretty sure it'll work. Will test immediately after merge.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Environment
+
+<!-- Use Help/About in Git Extensions to copy this information to the clipboard automatically. -->
+
+- GitExtensions version:
+- GIT version: 
+- OS version: 
+- .NET version: 
+
+
```
Comment (vbjay) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Diagnostics
<!--At your discretion, use tools found  on this [wiki](https://github.com/gitextensions/gitextensions/wiki/Diagnosing-problems) page-->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+<!--
+    Think through your proposal and describe it clearly.
+
+    Note that features are only added to the most recent version of Git Extensions.
+-->
+
+
+
+
```
Comment (gerhardol) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I suggest adding Environment here too, so we can see if a user have attempted to use any later version

Comment (drewnoakes) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Good idea. Fixed.

**Comment by drewnoakes** [comment]
@RussKie do you have any objections to this?

**Comment by RussKie** [comment]
No, please go ahead. Thank you

**Comment by vbjay** [comment]
Fyi I filed this with GitHub support. 

We have multiple issue templates in gitextensions/gitextensions repo. When on mobile site on the issues page and I tap + it gives me a blank issue form instead of letting me select template to use. 
screenshot_20190104-084340

**Comment by RussKie** [comment]
Send it to @githib and cc: @natfriedman on Twitter 😉

**Comment by vbjay** [comment]
:D  https://twitter.com/vbjaynet/status/1081293574925402113
LOL!

**Comment by RussKie** [comment]
It appears since we have new templates, we need to update links in the app to `https://github.com/gitextensions/gitextensions/issues/new/choose`

**Pull request merged**